casepro/backend/junebug.py

```diff
@@ -115,7 +188,28 @@ def pull_contacts(
             tuple of the number of contacts created, updated, deleted and
             ignored
         """
-        return (0, 0, 0, 0)
+        identity_store = self.identity_store
+
+        # all identities created in the Identity Store in the time window
+        new_identities = identity_store.get_identities(
+            created_at__gte=modified_after, created_at__lte=modified_before)
```

Comment:
The search endpoint isn't really for this. I would use the `/identities/` base and update https://github.com/praekelt/seed-identity-store/blob/develop/identities/views.py#L37 to have more of the fields you want to filter on